build/build.sh

```diff
@@ -20,7 +20,7 @@ done
 
 # This argument is required to read the latest kernel version from the netboot server
 if [[ "$netbootIP" == "" ]]; then
```

Comment:
I don't think, we need the netbootIP here anymore. We only use it as docker build argument. Though I'm not so sure, what the implications are, if we remove that argument.

So... can also be removed later and depricated now imo.

Reply:
As well as in the azure-pipelines.yaml